### DISCUSSION TITLE
[@types/video.js] VideoJsPlayerOptions is missing userActions.click property

### DISCUSSION THREAD

**Discussion by BrainCrumbz** [DefinitelyTyped/DefinitelyTyped#59685] [Issues with a @types package] [answered]
Hi all (again)

When setting options to create a video player, video.js [doc page](https://docs.videojs.com/tutorial-options.html#useractions) mentions a `userActions` property:

> userActions.click
> Type: boolean|function

So there seems to be one more property, apart from those already defined in [video.js/index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/be6aacc0ea7b39fdf44110887e1a489581f5ab1f/types/video.js/index.d.ts#L5796), `doubleClick` and `hotkeys`.

We stumbled upon this as we converted a JS project to TS, where we had code block similar to the following which is giving errors:

```ts
const options: VideoJsPlayerOptions = {
  // ...
  userActions: {
    click: false,
  },
};
```

HTH

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #59687
